testsuite/features/support/constants.rb

```diff
@@ -86,10 +88,10 @@
                       'sle15sp3_ssh_minion' => 'bison',
                       'sle15sp4_minion' => 'bison',
                       'sle15sp4_ssh_minion' => 'bison',
-                      'slemicro52_minion' => 'bison',
-                      'slemicro52_ssh_minion' => 'bison',
-                      'slemicro53_minion' => 'bison',
-                      'slemicro53_ssh_minion' => 'bison',
+                      'slemicro52_minion' => 'ethtool',
+                      'slemicro52_ssh_minion' => 'ethtool',
+                      'slemicro53_minion' => 'ethtool',
+                      'slemicro53_ssh_minion' => 'ethtool',
```

Comment:
We need a different packages fro SLE Micro since it does not have either `autoconf` nor `bison`.